docs/howto/openstack/nova/server-group.md

```diff
@@ -0,0 +1,46 @@
+---
+description: >
+  Server groups provide a mechanism to group servers
+  according to a certain policy.
+---
+# Using server groups
+
+{{page.meta.description}}
+
+In {{brand}}, you can use server groups to control the scheduling of a group of servers. A server group can have one of four different policies: affinity, soft-affinity, anti-affinity or soft-anti-affinity.
+
+## Prerequisites
+
+In order to use server groups you must use the OpenStack CLI, Make sure you have [enabled it](/howto/getting-started/enable-openstack-cli).
+
+## Affinity
+
+A server group with the policy of "affinity" will make sure that all the servers in that group are **always** placed on the same physical compute node.
+
+While a policy of "soft-affinity" will **try to** make sure that all the servers in that group are placed on the same physical compute node, but ultimately will allow it if otherwise not possible.
+
+## Anti-affinity
+
+A server group with the policy of "anti-affinity" will make sure that the servers in that group are **never** placed on the same physical compute node.
+
+> With "anti-affinity" you may also configure how many server you want to allow on the same physical compute node using, `--rule max_server_per_host=<number>`, where `<number>` is the amount of servers to allow on the same physical compute node.
+
+While a policy of "soft-anti-affinity", will **try to** make sure that all the servers in that group are not placed on the same physical compute node, but ultimately will allow it if otherwise not possible.
+
+## Creating server groups
+
+To create a server group, use the following command:
+
+```bash
+openstack server group create --policy [affinity|soft-affinity|anti-affinity|soft-anti-affinity] <server_group_name>
```

Comment:
With the enumeration of the available policies, this is now a bit too wide when rendering. So we might want to change this to:
```suggestion
openstack server group create \
  --policy [affinity|soft-affinity|anti-affinity|soft-anti-affinity] \
  <server_group_name>
```